### ISSUE TITLE
Fix sLSTM and mLSTM to work with 1 layer

### ISSUE THREAD

**Issue by incllude** [muditbhargava66/PyxLSTM#20]
sLSTM and mLSTM have dropout layers with length of 1 less than other layers:
```
self.dropout_layers = nn.ModuleList([nn.Dropout(dropout) for _ in range(num_layers - 1)])
```
So if you will try to run models with only one layer, no layers will be applied to input because of this for:
```
for i, (lstm, dropout, f_gate, i_gate) in enumerate(zip(self.lstms, self.dropout_layers, self.exp_forget_gates, self.exp_input_gates))
```

**Comment by Marathon19** [comment]
How to replace lstm with slstm when num_layers=1
